.gitignore

```diff
@@ -152,3 +152,6 @@ rush.json
 
 # tsdoc
 tsdoc-metadata.json
+
+#vscode workspace
```

Comment:
Should be this in "# Visual Studio Temp Project" section?
I have seen there `.idea` folder, too